cmd/buildah/push.go

```diff
@@ -148,6 +178,11 @@ func pushCmd(c *cobra.Command, args []string, iopts pushResults) error {
 		}
 	}
 
+	compress := imagebuildah.Gzip
+	if iopts.disableCompression {
+		compress = imagebuildah.Uncompressed
+	}
+
```

Comment:
hmm, might be better to do this in the calling function and pass it down to pushImage....

Reply:
moved.